include/javabind/chrono.hpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * javabind: effective C++ and Java interoperability
+ * @see https://github.com/hunyadi/javabind
+ *
+ * Copyright (c) 2024 Levente Hunyadi
+ *
+ * This work is licensed under the terms of the MIT license.
+ * For a copy, see <https://opensource.org/licenses/MIT>.
+ */
+#pragma once
+#include "object.hpp"
+#include "signature.hpp"
+#include <chrono>
+
+namespace javabind
+{
+    template <typename T>
+    struct JavaDurationTraits;
```

Comment:
I think we are missing the specialization for `std::chrono::microseconds`.

Java lacks `ofMicros` and `toMicros` on `time.Duration` but the missing methods can be emulated by calling `ofNanos` and `toNanos`, and applying proper scale.